test/e2e/cluster-logging/sd_soak.go

```diff
@@ -60,18 +60,25 @@ var _ = framework.KubeDescribe("Cluster level logging implemented by Stackdriver
 		podRunCount := int(testDuration.Seconds())/int(podRunDelay.Seconds()) - 1
 		linesPerPod := linesPerPodPerSecond * int(jobDuration.Seconds())
 
-		By("Running short-living pods")
 		pods := []*loggingPod{}
 		for runIdx := 0; runIdx < podRunCount; runIdx++ {
 			for nodeIdx, node := range nodes {
 				podName := fmt.Sprintf("job-logs-generator-%d-%d-%d-%d", maxPodCount, linesPerPod, runIdx, nodeIdx)
-				pods = append(pods, createLoggingPod(f, podName, node.Name, linesPerPod, jobDuration))
-
-				defer f.PodClient().Delete(podName, &meta_v1.DeleteOptions{})
+				pods = append(pods, newLoggingPod(podName, node.Name, linesPerPod, jobDuration))
 			}
-			time.Sleep(podRunDelay)
 		}
 
+		By("Running short-living pods")
+		go func() {
```

Comment:
Do you want pods themselves to run sequentially? I.e. is it ok that there is just one running at the same time?